### PR TITLE
[WFLY-15683] Update WildFly Preview to Jakarta Security 3.0.

### PR DESCRIPTION
**Pull request by darranl**
https://issues.redhat.com/browse/WFLY-15683
https://issues.redhat.com/browse/WFLY-16601

### CODE REVIEW

**Review by fjuma — APPROVED** [review]
Thanks @darranl!

**Comment by darranl** [comment]
Looks like a genuine failure on WildFly Preview

**Review by fjuma — APPROVED** [review]
Thanks @darranl.

File: ee-9/common/src/main/resources/modules/system/layers/base/com/nimbusds/nimbus-jose-jwt/main/module.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<module xmlns="urn:jboss:module:1.9" name="com.nimbusds.nimbus-jose-jwt">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${com.nimbusds:nimbus-jose-jwt}"/>
+    </resources>
+
+    <dependencies>
```
Comment (bstansberry) [inline ee-9/common/src/main/resources/modules/system/layers/base/com/nimbusds/nimbus-jose-jwt/main/module.xml:34]: Are any of the Bouncycastle or Google Tink deps needed?

https://repo1.maven.org/maven2/com/nimbusds/nimbus-jose-jwt/9.23/nimbus-jose-jwt-9.23.pom

Comment (bstansberry) [reply ee-9/common/src/main/resources/modules/system/layers/base/com/nimbusds/nimbus-jose-jwt/main/module.xml:34]: 663kb jar. ToT

**Comment by bstansberry** [comment]
Thanks @darranl. I merged this but I still have my question at https://github.com/wildfly/wildfly/pull/15750#discussion_r917199451

**Pull request merged**